Dockerfile

```diff
@@ -479,6 +479,7 @@ LABEL org.apache.airflow.distro="debian" \
   org.opencontainers.image.title="Production Airflow Image" \
   org.opencontainers.image.description="Reference, production-ready Apache Airflow image"
 
+ENV DUMB_INIT_SETSID="1"
 
```

Comment:
This is the "explicit is better than implicit" at it fullest. :) . 

By adding it here, anyone who gets the Dockerfile and modify it will be aware of the existence of that variable, and **might** start thinking what it is about, without actually reading the documentation. 

There are two kinds of engineers: those who read all the docs and those who 'just try and see' - this is more for the latter crowd.